ivy/container/activations.py

```diff
@@ -1,9 +1,147 @@
+# global
+from typing import Optional, Union, List, Dict, Tuple
+
 # local
+import ivy
 from ivy.container.base import ContainerBase
 
 # ToDo: implement all methods here as public instance methods
 
 
 # noinspection PyMissingConstructor
 class ContainerWithActivations(ContainerBase):
-    pass
+    def relu(
+        self: ivy.Container,
+        key_chains: Optional[Union[List[str], Dict[str, str]]] = None,
+        to_apply: bool = True,
+        prune_unapplied: bool = False,
+        map_sequences: bool = False,
+        out: Optional[ivy.Container] = None,
+    ) -> ivy.Container:
+        return self.handle_inplace(
```

Comment:
this should be calling the static method, please see the latest deep dive container section. Same for all other functions.

Reply:
Hey @kirodev, can you let me know after these changes are added as well ?

Reply:
Sure !